src/main/java/com/google/sps/data/Availability.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
```

Comment:
What is `sps`?

(I probably should've asked this elsewhere. I suspect this should be `sps.interviewme`, but I'm not sure..

Reply:
@acmcarther I thought that in case we need to use another class, the inclusion of the package allows us to access them

Reply:
I think this needs to be addressed across the project. You probably shouldn't be using the "sps" package (which I guess comes from the profile project), but rather some package dedicated for interview me. 

The best way to approach this would be to fix all of the files in a separate CL.

Reply:
Marking this resolved, sent a message in `STEP Team` about this.